chain/store/store.go

```diff
@@ -1165,6 +1172,24 @@ func (cs *ChainStore) GetTipsetByHeight(ctx context.Context, h abi.ChainEpoch, t
 	return cs.LoadTipSet(ctx, lbts.Parents())
 }
 
+func (cs *ChainStore) GetTipSetByCid(ctx context.Context, c cid.Cid) (*types.TipSet, error) {
```

Comment:
I kind of agree with @raulk here, as it would be dead code if we lifted it.
But ultimately it is @arajasek 's decision; master yushie your input is requestd here.

Reply:
Basically, I'm trying to reduce our final patch to "FEVM only" changes to make it easier to review.